app/model/Cached.scala

```diff
@@ -50,6 +49,8 @@ object NoCache {
 
 case class NoCache[A](action: Action[A]) extends Action[A] {
 
+  implicit lazy val executionContext: ExecutionContext = ExecutionContext.Implicits.global
```

Comment:
We shouldn't import the execution context here but rather leave it for each controller using this action to inject as appropriate. We might also need to make this a trait to do that since the case class won't be allowed to have abstract members?